src/Clara/lib/modules/LocaleManager.js

```diff
@@ -1,86 +0,0 @@
-/**
- * @file Locale manager.
```

Comment:
Revert this, we need the Locale manager

Reply:
Then the settings system is needed to use it properly.

Reply:
@Ovyerus some commands still uses unless we explicitly force english only at this point

Reply:
We can do that then.